### ISSUE TITLE
2020 11 04 A New Tab Does Not Open When Pressing on New Link.mp4

### ISSUE THREAD

**Issue by jill-iterators** [CityOfBoston/Iterators#192]
**I entered this before but I believe there was an internet Glitch as I do not see it. My apologies if it there but could not be found.**

[2020 11 04 A New link does not open when pressing on new link.mp4.zip](https://github.com/CityOfBoston/Iterators/files/5490235/2020.11.04.A.New.link.does.not.open.when.pressing.on.new.link.mp4.zip)

Environment
(Device > OS > Browser
Computer > MacOs 10.15.7 > Chrome 86.0

Trusted Tester Test

Brief Description
After opening private browser window and

 clicking on new link, a new window does not open for the new link, and tester loses focus. This is an Accessibility bug and a functional bug.

Steps to reproduce
1)	00.00, Paste the link noted above into a private browser window, press Return.
2)	00:00, New address is already populated into the Address field.
3)	00:16, Press on the link: the City’s Historic Districts
4)	00:20, Landmark Commission page appears in the same and both listed under Historic District do not open in a new tab that was previously opened. Note, this is an accessibility bug as the user can lose focus. Having the link open in a different window is a best practice that boston.gov follows. 

Desired Result
•	Able to click on a link and have a new window open so that Focus is not lost
Actual Result
•	Unable to stay on home page and have a new window open so that Focus is not lost

**Comment by subaha-cob** [comment]
@jill-iterators I am not sure what you mean here. The links are city of Boston links they are on the same domain and are all relative. The external links all open in a new window. Is there a reason links on the same domain should open in a new tab/window?

**Comment by reillyzlab** [comment]
Won't implement this, so closing.
